lib/OpenQA/Git.pm

```diff
@@ -101,7 +103,7 @@ sub commit ($self, $args = undef) {
         return undef if $res->{status};
 
         my $msg = 'Unable to push Git commit';
-        if ($res->{return_code} == 128 and $res->{stderr} =~ m/Authentication failed for .http/) {
+        if ($res->{return_code} == 1 and $res->{stderr} =~ m/Authentication failed for .http/) {
```

Comment:
I'm sorry that I previously was confusing. But the return code here is 32768, so that would be 128 with `>> 8`

Reply:
Interesting. I provoked an auth error via SSH and got a `1` return code. However, now I provoked one via HTTP and got `128`. So apparently it depends on the protocol. I'll change it back then.